src/daisy_patch_sm.cpp

```diff
@@ -6,51 +6,51 @@ namespace daisy
 namespace patch_sm
 {
     /** Const definitions */
-    static constexpr dsy_gpio_pin DUMMYPIN        = {DSY_GPIOX, 0};
-    static constexpr dsy_gpio_pin PIN_ADC_CTRL_1  = {DSY_GPIOA, 3};
-    static constexpr dsy_gpio_pin PIN_ADC_CTRL_2  = {DSY_GPIOA, 6};
-    static constexpr dsy_gpio_pin PIN_ADC_CTRL_3  = {DSY_GPIOA, 2};
-    static constexpr dsy_gpio_pin PIN_ADC_CTRL_4  = {DSY_GPIOA, 7};
-    static constexpr dsy_gpio_pin PIN_ADC_CTRL_5  = {DSY_GPIOB, 1};
-    static constexpr dsy_gpio_pin PIN_ADC_CTRL_6  = {DSY_GPIOC, 4};
-    static constexpr dsy_gpio_pin PIN_ADC_CTRL_7  = {DSY_GPIOC, 0};
-    static constexpr dsy_gpio_pin PIN_ADC_CTRL_8  = {DSY_GPIOC, 1};
-    static constexpr dsy_gpio_pin PIN_ADC_CTRL_9  = {DSY_GPIOA, 1};
-    static constexpr dsy_gpio_pin PIN_ADC_CTRL_10 = {DSY_GPIOA, 0};
-    static constexpr dsy_gpio_pin PIN_ADC_CTRL_11 = {DSY_GPIOC, 3};
-    static constexpr dsy_gpio_pin PIN_ADC_CTRL_12 = {DSY_GPIOC, 2};
-    static constexpr dsy_gpio_pin PIN_USER_LED    = {DSY_GPIOC, 7};
-
-    const dsy_gpio_pin kPinMap[4][10] = {
+    static constexpr Pin DUMMYPIN        = Pin(); //Pin(PORTX, 0);
+    static constexpr Pin PIN_ADC_CTRL_1  = Pin(PORTA, 3);
+    static constexpr Pin PIN_ADC_CTRL_2  = Pin(PORTA, 6);
+    static constexpr Pin PIN_ADC_CTRL_3  = Pin(PORTA, 2);
+    static constexpr Pin PIN_ADC_CTRL_4  = Pin(PORTA, 7);
+    static constexpr Pin PIN_ADC_CTRL_5  = Pin(PORTB, 1);
+    static constexpr Pin PIN_ADC_CTRL_6  = Pin(PORTC, 4);
+    static constexpr Pin PIN_ADC_CTRL_7  = Pin(PORTC, 0);
+    static constexpr Pin PIN_ADC_CTRL_8  = Pin(PORTC, 1);
+    static constexpr Pin PIN_ADC_CTRL_9  = Pin(PORTA, 1);
+    static constexpr Pin PIN_ADC_CTRL_10 = Pin(PORTA, 0);
+    static constexpr Pin PIN_ADC_CTRL_11 = Pin(PORTC, 3);
+    static constexpr Pin PIN_ADC_CTRL_12 = Pin(PORTC, 2);
+    static constexpr Pin PIN_USER_LED    = Pin(PORTC, 7);
+
+    const Pin kPinMap[4][10] = {
         /** Header Bank A */
         {
-            DUMMYPIN,        /**< A1  - -12V Power Input */
-            {DSY_GPIOA, 1},  /**< A2  - UART1 Rx */
-            {DSY_GPIOA, 0},  /**< A3  - UART1 Tx */
-            DUMMYPIN,        /**< A4  - GND */
-            DUMMYPIN,        /**< A5  - +12V Power Input */
-            DUMMYPIN,        /**< A6  - +5V Power Output */
-            DUMMYPIN,        /**< A7  - GND */
-            {DSY_GPIOB, 14}, /**< A8  - USB DM */
-            {DSY_GPIOB, 15}, /**< A9  - USB DP */
-            DUMMYPIN,        /**< A10 - +3V3 Power Output */
+            DUMMYPIN,       /**< A1  - -12V Power Input */
+            Pin(PORTA, 10), /**< A2  - UART1 Rx */
```

Comment:
I think we bypassed our full testing on the PatchSM/Patch.Init earlier in the process (when we were talking about it duplicating the efforts in the existing PR).

Just noticed that this pin got converted incorrectly; it should be `Pin(PORTA, 1)`

We'll go through a quick test on the patch to see if there's anything else that we missed, but this looks like the only issue.